domain/src/main/java/no/difi/meldingsutveksling/core/EDUCore.java

```diff
@@ -16,20 +17,25 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 @XmlType(name = "EDUCore", propOrder = {
         "id",
+        "messageReference",
         "sender",
         "receiver",
+        "serviceIdentifier",
         "messageType",
         "payload"
 })
 public class EDUCore {
 
-    // ConversationId
     @XmlAttribute(name = "id", required = true)
     private String id;
+    @XmlElement
+    private String messageReference;
     @XmlElement(required = true)
     private Sender sender;
     @XmlElement(required = true)
     private Receiver receiver;
+    @XmlElement
+    private ServiceIdentifier serviceIdentifier; // TODO: remove when MessageStrategy is Springified (due to ReceiptRepo..)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)